installer/Migration/V5_0_0/lang-string/general.yaml

```diff
@@ -845,3 +845,18 @@ langStrings:
   -
     value: 'Successfully deleted'
     unitId: successfully_deleted
+  -
```

Comment:
@SharukaNP please let me know if the upgrader tool you're working on is available for these lang strings to be able to test by QA